webapp/src/main/webapp/skin/less/news.less

```diff
@@ -590,118 +590,12 @@
     }
   }
 
-  .newsComposerActions {
+  .newsComposerToolbar {
```

Comment:
we need to verify that all references to publish/unpublish except publishing drawer are cleaned for news composer